new/LIT7080Qene.xml

```diff
@@ -0,0 +1,79 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7080Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ወሞት፡ መንፈቀ፡ ስጋነ፡ ለረኃብ፡ ወመንፈቀ፡ ሥጋነ፡ ለመ<choice resp="JK"><sic>ሩ</sic><corr>ሬ</corr></choice>ት፤</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Wa-mot manfaqa sǝgāna la-raḫāb wa-manfaqa śǝgāna la-maret (Qǝne of the śəllāse-type)</title>
+                <title xml:lang="en" corresp="#t1">And death, half of our flesh for hunger and half of our flesh for the soil…</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="JK"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="UppQ1524o"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><foreign xml:lang="gez">Śəllāse</foreign>-type <foreign xml:lang="gez">qǝne</foreign> attributed to 
+                    <persName ref="PRS14574GabraLeul" cert="high" corresp="SD">Gabra Ləʿul</persName> 
+                    in the manuscript. In the manuscript, it appears that the first word of the first line was originally (in the Vorlage?)
+                    intended as a <foreign xml:lang="gez">qǝne</foreign>-type indication.</p>
```

Comment:
The second sentence is a little complicated and hypothetic. Better: "The <foreign xml:lang="gez">qǝne</foreign>-type is indicated in the beginning of the first line.